mesonbuild/utils/universal.py

```diff
@@ -683,14 +682,8 @@ def is_qnx() -> bool:
 def is_aix() -> bool:
     return platform.system().lower() == 'aix'
 
-def exe_exists(arglist: T.List[str]) -> bool:
-    try:
-        if subprocess.run(arglist, timeout=10).returncode == 0:
-            return True
-    except (FileNotFoundError, subprocess.TimeoutExpired):
-        pass
-    return False
-
+def exe_exists(exe: str) -> bool:
+    return shutil.which(exe) is not None
```

Comment:
I thought we were removing this entirely?

Reply:
Created https://github.com/mesonbuild/meson/pull/13456 to remove the function entirely.